packages/objects/src/interfaces/IConsentBaseURISchema.ts

```diff
@@ -0,0 +1,8 @@
+import { IUserAgreement } from "@objects/interfaces/IUserAgreement.js";
+import { URLString } from "@objects/primitives/URLString.js";
+
+export interface IConsentBaseURISchema extends IUserAgreement {
```

Comment:
We could change the IUserAgreement and it’s name, rather than extending it.

Reply:
agreed